closure/goog/editor/plugins/enterhandler.js

```diff
@@ -631,6 +631,22 @@ goog.editor.plugins.EnterHandler.prototype.deleteCursorSelectionW3C_ =
   return goog.editor.range.getDeepEndPoint(range, true);
 };
 
+/**
+ * Checks Whether the selection range start from beginning.
+ * @param {Node} node the element or text node the range starts in.
+ * @param {Node} baseNode the container element.
+ * @return {boolean} Whether the selection range start from beginning.
+ * @private
+ */
+goog.editor.plugins.EnterHandler.isStartFromBeginning_ = function(node, baseNode) {
```

Comment:
This method took me a bit to understand - can we think of a better name here that is more descriptive? Maybe isNodeLeftMostChild ?

Reply:
OK!! I'll rename it to `isNodeLeftMostChild `.

Reply:
094d698